src/dists.ad.js

```diff
@@ -505,15 +778,15 @@ function expGammaSample(shape, scale) {
 function expGammaScore(shape, scale, val) {
   'use ad';
   var x = val;
-  return (shape - 1) * x - Math.exp(x) / scale - logGamma(shape) - shape * Math.log(scale);
+  return (shape - 1) * x - Math.exp(x) / scale - Math.logGamma(shape) - shape * Math.log(scale);
```

Comment:
This is surprising behavior for functions that are in `ad.scalar.*` but not in (base) `Math.*`.